signup.html

```diff
@@ -1,59 +1,64 @@
 <!DOCTYPE html>
 <html lang="ko">
 <head>
-    <meta charset="UTF-8">
-    <title>signup</title>
-    <link rel="stylesheet" href="css/signin.css">
-    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta charset="UTF-8">
+  <title>signup</title>
+  <link rel="stylesheet" href="css/signin.css">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
 </head>
 <body>
   <div class="contents">
     <div class="title">
       <div class="logo">
         <a href="/"> 
-          <img src="images/logo.svg">
+          <img src="images/logo.svg" alt="로고 이미지">
         </a>
       </div>
       <h2>
-        이미 회원이신가요>
+        이미 회원이신가요?
         <a href="./signin.html">로그인하기</a>
       </h2>
     </div>
     <div class="input-form">
-      <div class="email-input">
-        <h3>이메일</h3>
-        <form>
-          <input type="email">
-        </form>
-      </div>
-      <div class="pwd-input">
-        <h3>비밀번호</h3>
-        <form>
-          <input name="focus" type="password">
-        </form>
-      </div>
-      <div class="pwd-input-check">
-        <h3>비밀번호 확인</h3>
-        <form style="margin-bottom: 12px;">
-          <input name="focus" type="password">
-        </form>
-        <p> 비밀번호와 다릅니다</p>
-      </div>
+      <form>
+        <ol>
+          <li>
+            <label>
+              이메일 <br>
```

Comment:
의도하지 않은 여백 삭제해주세요! 
37, 43, 46 라인 등 같이 확인해주세요 

```suggestion
              이메일<br>
```